daemon/gamemode.c

```diff
@@ -604,6 +605,78 @@ GameModeContext *game_mode_context_instance()
 	return &instance;
 }
 
+/**
+ * Attempt to resolve the exe for wine-preloader.
+ * This function is used if game_mode_context_find_exe() identified the
+ * process as wine-preloader.
+ */
+static char *game_mode_resolve_wine_preloader(pid_t pid)
+{
+	static char proc_path[PATH_MAX] = { 0 };
+	int fd = 0, proc_fd = 0;
```

Comment:
0 is a valid FD, even if it is usually taken by stdin. Better to initialise to -1 and change the cleanup code accordingly.

Reply:
Good point, will do.

Reply:
Fixed.